.github/workflows/test-prs.yml

```diff
@@ -23,4 +23,27 @@ jobs:
                   ${{ runner.os }}-node-
             - run: npm install
             - run: npm run lint
-            - run: npm test
+            - run: npm run test
+    addCoverage:
```

Comment:
Does the `addCoverage` step need to do all the setup again? Is it possible to replace `npm run test` with `npm run test:coverage` and then add `Upload coverage to Codecov` as the next step?

Reply:
Now looking in the file again, we do not need to perform the setup again. If the tests fail, we will not upload the coverage to Codecov.
Adjusted properly to optimize workflow.